pages/golfers/[id].js

```diff
@@ -0,0 +1,62 @@
+import { useRouter } from 'next/router'
+import useSWR from 'swr'
+import Layout from '../../components/Layout'
+import ScoreCard from '../../components/ScoreCard'
+
+
+const Profile = () => {
+  const fetcher = async url => {
+    const response = await fetch(url, {
+      method: 'GET',
+    })
+
+    if (!response.ok) {
+      const error = new Error('An error occurred while fetching the data.')
+      error.info = await response.json()
+      error.status = response.status
+      throw error
+    }
+
+    return response.json()
+  }
+
+  const router = useRouter()
+  const { id } = router.query
+  const PROFILE_URL = `${process.env.NEXT_PUBLIC_API_URL}/golfers/${id}`
+  const { data, error } = useSWR(PROFILE_URL, fetcher) // data = { name: ..., scores: ...}
+
+  return (
+    <Layout>
+      <>
+        {error ? (
+          error.message
+        ) : (
+          <>
+            {
+              data
+                ? <span className="username">{data.name}</span>
+                : null
+            }
```

Comment:
In this case, if the "else" body of the ternary operator returns `null` or `undefined`, we can use a shorter syntax
```js
{data && <span className="username">{data.name}</span>}
```
or when the line gets long
```js
{data && (
  <span className="username">{data.name}</span>
)}
```